app/services/listing_index_service/api/api.rb

```diff
@@ -7,7 +7,7 @@ class << self
     end
 
     def self.listings
-      self.listings_api ||= ListingIndexService::API::Listings.new
+      self.listings_api ||= ListingIndexService::API::Listings.new(Rails.logger)
```

Comment:
kewl, I was thinking about that but didn't realize it's already in master. Will definitely change.